feature/visited/src/main/kotlin/uk/govuk/app/visited/ui/widget/VisitedWidget.kt

```diff
@@ -24,10 +24,12 @@ import uk.govuk.app.design.ui.theme.GovUkTheme
 
 @Composable
 fun VisitedWidget(
+    onClick: (String) -> Unit,
     modifier: Modifier = Modifier
 ) {
     Column(modifier = modifier) {
         OutlinedCard(
+            onClick = { onClick("Visited") },
```

Comment:
Do we need to pass the actual text on the widget here?

Reply:
We can - have updated 👍